Portfolio/src/App.vue

```diff
@@ -3,96 +3,95 @@ import { ref } from 'vue';
-
-            <nav>
-                <ul class="navbar">
-                    <li>
-                        <RouterLink style="text-decoration: none; color: #1e1e1e; ;" to="/">Home
-                        </RouterLink>
-                    </li>
-                    <li>
-                        <RouterLink style="text-decoration: none; color: #1e1e1e; " to="/work">Work</RouterLink>
-                    </li>
-                    <li>
-                        <RouterLink style="text-decoration: none; color: #1e1e1e;" to="/about">About</RouterLink>
-                    </li>
-                    <li>
-                        <RouterLink style="text-decoration: none; color: #1e1e1e;" to="/contact">Contact</RouterLink>
-                    </li>
-                </ul>
-            </nav>
-
-        </div>
-    </header>
-
-    <div v-if="isMenuShown" class="openedMenu" style="font-family: 'RalewayBold'; text-transform: uppercase;">
-        <ul class="menu-links">
-            <li>
-                <RouterLink style="text-decoration: none; color: #ffff; ;" to="/">Home
-                </RouterLink>
-            </li>
-            <li>
-                <RouterLink style="text-decoration: none; color: #ffff; " to="/work">Work</RouterLink>
-            </li>
-            <li>
-                <RouterLink style="text-decoration: none; color: #ffff;" to="/about">About</RouterLink>
-            </li>
-            <li>
-                <RouterLink style="text-decoration: none; color: #ffff;" to="/contact">Contact</RouterLink>
-            </li>
-        </ul>
-
-        <div class="menu-fullName">
-            <h1 class="menu-firstName">{{ homeViewContent.firstName }}</h1>
-            <h1 class="menu-lastName">{{ homeViewContent.lastName }}</h1>
+    <Wrapper>
+        <header>
+            <div class="router-wrapper">
+                <button @click="isMenuShown = !isMenuShown" class="menuButton">
+                    <div class="line" :class="{ 'whiteLine': isMenuShown }">
+                    </div>
+                    <div class="line" :class="{ 'whiteLine': isMenuShown }"></div>
+                    <div class="line" :class="{ 'whiteLine': isMenuShown }"></div>
+                </button>
+
+                <nav>
+                    <ul class="navbar">
+                        <li>
+                            <RouterLink style="text-decoration: none; color: #1e1e1e; ;" to="/">Home
+                            </RouterLink>
+                        </li>
+                        <li>
+                            <RouterLink style="text-decoration: none; color: #1e1e1e; " to="/work">Work</RouterLink>
+                        </li>
+                        <li>
+                            <RouterLink style="text-decoration: none; color: #1e1e1e;" to="/about">About</RouterLink>
+                        </li>
+                        <li>
+                            <RouterLink style="text-decoration: none; color: #1e1e1e;" to="/contact">Contact
```

Comment:
We would always define our routes in the route table and refer to the routes using variables. Hardcoded values are a very bad idea, because your application becomes very hard to maintain as it grows. This is a simple portfolio website, however the point still stands.

Reply:
I will pair-code with @Evgenios95 and raise a separate PR for it.